packages/server/types/index.d.ts

```diff
@@ -16,11 +16,11 @@ import * as Knex from 'knex'
 import * as Koa from 'koa'
 import koaBodyParser from 'koa-bodyparser'
 import koaCompress from 'koa-compress'
-import koaLogger from 'koa-logger'
-import mount from 'koa-mount'
-import koaPinoLogger from 'koa-pino-logger'
-import koaResponseTime from 'koa-response-time'
+import koaMount from 'koa-mount'
 import koaSession from 'koa-session'
+import koaResponseTime from 'koa-response-time'
+import koaLogger from '@types/koa-logger'
+import koaPinoLogger from '@types/koa-pino-logger'
```

Comment:
@puckey the linter was complaining about these imports, I hope I fixed it correctly?